File: generated-sources/kotlin-spring/mojang-authentication/src/main/kotlin/org/openapitools/model/Authentication.kt
package org.openapitools.model

import java.util.Objects
import org.openapitools.model.AccessKeys
import javax.validation.Valid
import javax.validation.constraints.*

/**
 * 
 */
data class Authentication (

        @get:NotNull 
        @JsonProperty("accessToken") override val accessToken: String,

        @JsonProperty("clientToken") override val clientToken: String? = null
) : AccessKeys{

}

File: generated-sources/scala-gatling/mojang-authentication/src/gatling/scala/org/openapitools/client/model/Authentication.scala

package org.openapitools.client.model


case class Authentication (
    _accessToken: String,
    _clientToken: Option[String]
)
object Authentication {
    def toStringBody(var_accessToken: Object, var_clientToken: Object) =
        s"""
        | {
        | "accessToken":$var_accessToken,"clientToken":$var_clientToken
        | }
        """.stripMargin
}
